File: cpp/program/makanan.cpp
#pragma once
#include <bits/stdc++.h>
using namespace std;

class makanan
{
private:
    string nama;
    string jenis;
public:
    // constructor
    makanan() {}
    makanan(string nama, string jenis) 
    {
        this->nama = nama;
        this->jenis = jenis;
    }

    // Setter
    void setNama(string nama) { this->nama = nama; }
    void setJenis(string jenis) { this->jenis = jenis; }

    // Getter
    string getNama() { return nama; }
    string getJenis() { return jenis; }

    void print() // method untuk menampilkan data makanan
    {
        cout << "Makanan            : " << getNama() << " (" << getJenis() << ")" << endl;
    }

    // Destructor
    ~makanan() {}
};

File: py/program/makanan.py
class makanan:
    # Constructor
    def __init__(self, nama="", jenis=""):
        self.nama = nama
        self.jenis = jenis

    # Setter
    def set_nama(self, nama):
        self.nama = nama

    def set_jenis(self, jenis):
        self.jenis = jenis

    # Getter
    def get_nama(self):
        return self.nama

    def get_jenis(self):
        return self.jenis

    # Method print
    def print(self):
        print(f"Makanan            : {self.get_nama()} ({self.get_jenis()})")

    # Destructor
    def __del__(self):
        pass